gdal/ci/travis/gcc48_stdcpp11/install.sh

```diff
@@ -18,10 +18,10 @@ cat java.opt | sed "s/JAVA_HOME =.*/JAVA_HOME = \/usr\/lib\/jvm\/java-7-openjdk-
 mv java.opt.tmp java.opt
 make
 cd ../..
-cd swig/perl
-make generate
-make
-cd ../..
+#cd swig/perl
+#CC=g++-4.8 make generate
```

Comment:
@ajolma would there be a way of making this work (ie forcing an alternate compiler than the default one) ?

Reply:
Probably Makefile.PL should be edited to pick the selected compiler from GDALmake.opt. Is that CXX (without the $(LIBTOOL_COMPILE_CXX)? And put it into the call to WriteMakefile as "CC => $compiler,". I'm already going through GDALmake.opt in Makefile.PL to get the man dirs.